distributed/kube_fitness/kube_fitness/tm.py

```diff
@@ -336,8 +336,9 @@ def fit_tm_of_individual(dataset: str,
             }
         except SoftTimeLimitExceeded as ex:
             raise ex
-        except Exception:
-            logger.exception(msg="Fitness calculation problem")
+        except Exception as e:
+            logger.error("Fitness calculation problem")
```

Comment:
logger.error has 'exc_info'.

if 'exc_info' is True, the logger will add info about the current exception.

So, no need for separate 'logger.exception' call.